azurerm/internal/services/cognitive/resource_arm_cognitive_account.go

```diff
@@ -74,6 +74,7 @@ func resourceArmCognitiveAccount() *schema.Resource {
 					"CustomVision.Training",
 					"Emotion",
 					"Face",
+					"FormRecognizer",
```

Comment:
could we also [add this to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/cognitive_account.html.markdown#L45)?

Reply:
Done